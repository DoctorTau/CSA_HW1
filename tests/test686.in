51
-8396 -5915 351 5012 -5389 2852 5144 -5929 -9526 -2158 -3265 1291 3995 1162 -5356 7445 5529 -1566 7097 6461 5326 9869 5104 -1984 -3185 1219 -5096 8417 -6218 -7729 4356 -632 6581 6338 331 2196 1157 85 6701 -2089 3642 820 1920 9046 4788 -5356 -3006 -3565 5141 -2783 1322 