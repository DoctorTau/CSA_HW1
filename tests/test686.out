-8396 -9526 -2158 -5356 -1566 -1984 -5096 -6218 -632 -5356 -3006 